From ae43cd883313e2309b3f90416b69e2e008b54a25 Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Thu, 1 Oct 2020 11:16:02 +0000
Subject: [PATCH] Revert "krdb: Call xrdb with -nocpp" to fix gitk runtime
 errors

This commit was preventing gitk from starting with
'unknown color name "BACKGROUND"' errors.

This reverts commit d360e3dcb608bd96d674bbae597b7938334972fd.


(cherry picked from commit f0764d87fe5275a123cdf6dca1ab3d0399c97578)
---
 kcms/krdb/krdb.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/kcms/krdb/krdb.cpp b/kcms/krdb/krdb.cpp
index c47e0353b..59ba546c4 100644
--- a/kcms/krdb/krdb.cpp
+++ b/kcms/krdb/krdb.cpp
@@ -493,11 +493,11 @@ void runRdb( uint flags )
   tmpFile.flush();
 
   KProcess proc;
-  proc << QStringLiteral("xrdb") << QStringLiteral("-nocpp")
-#ifdef NDEBUG
-    << QStringLiteral("-quiet")
+#ifndef NDEBUG
+  proc << QStringLiteral("xrdb") << QStringLiteral("-merge") << tmpFile.fileName();
+#else
+  proc << "xrdb" << "-quiet" << "-merge" << tmpFile.fileName();
 #endif
-    << QStringLiteral("-merge") << tmpFile.fileName();
   proc.execute();
 
   applyGtkStyles(exportColors, 1);
-- 
2.25.1

